website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -100,50 +61,7 @@ In Figma, the `critical` Modal color is coupled with the `critical` Button color
 
 We recommend using the **medium** size Modal for most scenarios, but use the size that best accounts for the complexity of the content and intended speed of interaction. We do not recommend resizing the Modal manually.
 
-<Hds::Modal @size="small" id="size-small-modal-first" class="doc-modal-demo" open as |M|>
-  <M.Header>
-    Small modal
-  </M.Header>
-  <M.Body>
-    <p class="hds-typography-body-300 hds-foreground-primary">Modal content</p>
-  </M.Body>
-  <M.Footer>
-    <Hds::ButtonSet>
-      <Hds::Button type="submit" @text="Confirm" />
-      <Hds::Button type="button" @text="Cancel" @color="secondary" />
-    </Hds::ButtonSet>
-  </M.Footer>
-</Hds::Modal>
-
-<Hds::Modal @size="medium" id="size-medium-modal" class="doc-modal-demo" open as |M|>
-  <M.Header>
-    Medium modal
-  </M.Header>
-  <M.Body>
-    <p class="hds-typography-body-300 hds-foreground-primary">Modal content</p>
-  </M.Body>
-  <M.Footer>
-    <Hds::ButtonSet>
-      <Hds::Button type="submit" @text="Confirm" />
-      <Hds::Button type="button" @text="Cancel" @color="secondary" />
-    </Hds::ButtonSet>
-  </M.Footer>
-</Hds::Modal>
-
-<Hds::Modal @size="large" id="size-large-modal" class="doc-modal-demo" open as |M|>
-  <M.Header>
-    Large modal
-  </M.Header>
-  <M.Body>
-    <p class="hds-typography-body-300 hds-foreground-primary">Modal content</p>
-  </M.Body>
-  <M.Footer>
-    <Hds::ButtonSet>
-      <Hds::Button type="submit" @text="Confirm" />
-      <Hds::Button type="button" @text="Cancel" @color="secondary" />
-    </Hds::ButtonSet>
-  </M.Footer>
-</Hds::Modal>
+![A small, medium, and large modal vertically stacked.](/assets/components/modal/modal-sizes.png)
```

Comment:
[question] This is the only image with an alt text, any reason why this is different from the others? (I see the equivalent image for the Flyout doesn't, for example)

Reply:
I only put alt text for this one because the content of the page doesn't say what sizes of Modal are available... I should probably put alt text for the color variants to describe what is different about each of them.

Reply:
This is a good point.. I added more informative alt text for all the images.